core/frontend-devtools/src/tools/DisplayStyleTools.ts

```diff
@@ -172,8 +171,7 @@ export class SaveRenderingStyleTool extends DisplayStyleTool {
 
     this._options.includeAll = getArg("a");
     this._options.includeIModelSpecific = getArg("i");
-    // SWB What does project mean here?
-    this._options.includeProjectSpecific = getArg("p");
+    this._options.includeITwinSpecific = getArg("t");

```

Comment:
Is this arg change breaking? What else is supposed to change to be compatible

Reply:
Yes it will be. I think we need to update the docs for this.

cc: @pmconne

Reply:
Please leave it alone. There's already an "i" arg ("iModel") which I assume is why you went with "t" (for "twin"?) That's not a helpful change.

Reply:
Returned to using `"p"`.